benchmark/src/main/java/com/distkv/drpc/benchmark/common/common_pb.proto

```diff
@@ -0,0 +1,14 @@
+syntax="proto3";
```

Comment:
Do not define this file. Just put the context to `string_pb.proto` and rename `string_pb.proto` to `str.proto`